packages/wattpm-utils/bin/cli.js

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env node
+
+import { checkNodeVersionForServices, setExecutableId, setExecutableName } from '@platformatic/foundation'
+
+checkNodeVersionForServices()
+setExecutableId(process.env.WATTPM_EXECUTABLE_ID ?? 'wattpm-utils')
+setExecutableName(process.env.WATTPM_EXECUTABLE_NAME ?? 'Watt Utils')
```

Comment:
Why can this be configured via environment variables?

Reply:
They were only used in now useless test. Removed.